src/style.css

```diff
@@ -106,9 +106,19 @@ input:focus {
 }
 
 .todo-footer {
-  background-color: #f7f7f7;
+  background: rgb(209, 123, 123);
   text-align: center;
   padding: 16px;
+  font-weight: 900;
```

Comment:
- [ ] [OPTIONAL] I would like to strongly recommend you make sure  Your task is to build a website that displays a list that looks and behaves like the part of the minimalist project captured in the video below. :arrow_down:  to give a good user experience. Please fix the design :+1: 

![68747470733a2f2f696d672e796f75747562652e636f6d2f76692f416355642d5f596a6a71672f302e6a7067](https://user-images.githubusercontent.com/6615394/214983824-87f6efef-bc6b-4ac7-b468-0d07d4496b62.jpeg)

##### Your design
![Screenshot_20230127_005936(1)](https://user-images.githubusercontent.com/6615394/214984147-f62b2383-8a17-4ce7-a272-9176a1111726.jpg)